target/linux/ipq807x/base-files/etc/board.d/01_leds

```diff
@@ -21,6 +21,15 @@ qnap,301w)
 	ucidef_set_led_netdev "10G_1" "10G_1" "green:10g_1" "eth4"
 	ucidef_set_led_netdev "10G_2" "10G_2" "green:10g_2" "eth5"
 	;;
+netgear,rax120)
+	ucidef_set_led_default "power" "POWER" "white:power" "1"
+	ucidef_set_led_default "wps" "WPS" "white:wps" "1"
```

Comment:
LED default state can and should be set in the DTS.

Reply:
Thanks! Corrections pending.